HULKs/hulk
# Too early replacment keeper switch

## Introduced Changes

Prevents to early Keeper switches
Improves replacmentkeeper_test.lua file

Fixes #395 

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)

None

## How to Test

Penalize the keeper and unpenalize him afterwards and check if the switch is delayed
The upgraded lua file can be used for this testing
